### PR TITLE
 setup: remove ubuntu1404

### PR DESCRIPTION
**Pull request by maclover7**
I am able to successfully run `ansible/playbooks/jenkins/worker/create.yml`
on all Ubuntu 14.04 test machines, so I think this directory is now obsolete.

### CODE REVIEW

File: ansible/roles/java-base/tasks/main.yml [hunk 1]
```diff
@@ -34,11 +34,11 @@
 # if this fails you want to check in vars/main.yml and add package name
 # as appropriate -- try to use generic os family if available.
 - name: install java
-  when: not os|startswith("zos")
+  when: not os|startswith("zos") and "ppc64" not in ansible_nodename
   package: name="{{ java_package_name }}" state=present
 
 - name: install webupd8 oracle java 8 extras
-  when: os in ("ubuntu1204", "ubuntu1404")
+  when: os in ("ubuntu1204", "ubuntu1404") and "ppc64" not in ansible_nodename
```
Comment (mhdawson) [inline ansible/roles/java-base/tasks/main.yml]: I think @jbarz has a PR (or at least one in progress) that does the right thing for ppc in respect to installing java8

File: ansible/roles/java-base/tasks/main.yml [hunk 2]
```diff
@@ -34,11 +34,11 @@
 # if this fails you want to check in vars/main.yml and add package name
 # as appropriate -- try to use generic os family if available.
 - name: install java
-  when: not os|startswith("zos")
+  when: not os|startswith("zos") and "ppc64" not in ansible_nodename
```
Comment (mhdawson) [inline ansible/roles/java-base/tasks/main.yml]: This just seems to not install the packages on the ppc64 machines I don't think that is what we want?

**Comment by mhdawson** [comment]
@jbarz can you coordinate with @maclover7 I'd like to wait to remove the old scripts until we have the new ones updated with the right pieces for PPC.

**Comment by mhdawson** [comment]
This is the PR that @jbarz is working on https://github.com/nodejs/build/pull/1075

File: ansible/inventory.yml [hunk 3]
```diff
@@ -97,6 +97,7 @@ hosts:
         ubuntu1604-x86-1: {ip: 159.203.77.233}
         ubuntu1604_docker-x64-1: {ip: 128.199.198.56}
         ubuntu1604_docker-x64-2: {ip: 138.68.241.115}
+        ubuntu1404-x86-2: {ip: 127.0.0.1, port: 2222, user: vagrant}
```
Comment (rvagg) [inline ansible/inventory.yml]: remove this

**Comment by rvagg** [comment]
lgtm other than the nit in inventory.yml

**Comment by mhdawson** [comment]
I think we should just close this PR as the changes that were needed were part of the PR from @jBarz  that we needed.  We still have more work to do before ubuntu 14 can be removed.

**Comment by maclover7** [comment]
@mhdawson Just rebased, I think this can land now that #1075, #1103, and #1338 are in effect. I believe that PPC machines were the only edge case holding this back.

**Comment by maclover7** [comment]
Re-ran playbook once more to be sure, and everything seems to be okay

**Comment by maclover7** [comment]
Landed in a098c6d

**Pull request merged**